web/sites/default/config/core.entity_view_display.group.elected_official.default.yml

```diff
@@ -64,14 +64,14 @@ third_party_settings:
         - field_person_image
       meta:
         - field_summary
+        - field_contact
```

Comment:
I'll leave a note in the Jira issue, but we need to lead contact in the contact region to keep the fix we did for electeds with more than one contact.

Reply:
Good catch. Just added the fix.